### DISCUSSION TITLE
deleted table is converted to a folder on data-model view

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#14155] [Feature Requests]
### Describe the Bug

When deleting a table outside of directus, the `data-model` view displays it as a folder which it is not. Perhaps if the collection is not found in the database, then it can be safely assumed to be deleted (and the subsequent data removed from relevant directus tables) instead of defaulting to a folder. 

### To Reproduce

as above

### What version of Directus are you using?

9.12.2

### What version of Node.js are you using?

12.22.5

### What database are you using?

mariadb

### What browser are you using?

firefox

### How are you deploying Directus?

local

**Comment by br41nslug** [comment]
This is expected behavior because that is what "folders" are currently (namely collections without an actual table) so we can't assume collections not in the database have been deleted. If deleting tables outside of directus it is advised to also remove the row from `directus_collections`, its fields from `directus_fields` and its relations from `directus_relations` (possibly revisions too if needed).

**Reply by sxsws** [reply]
Sure, but with a database-first approach, it should know how to deal with such situations

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
